### PR TITLE
Creates Templates for feedback emailed to users

### PR DESCRIPTION
**Pull request by GrantComm**

### CODE REVIEW

**Review by meagancheese — APPROVED** [review]
LGTM

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Thank your for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below.
```
Comment (apluscs) [inline src/main/resources/templates/email/feedbackToInterviewee.txt]: Nit: your --> you, same for the other file.

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Thank your for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
+
+The values 1-10 represent whether your interviewer strongly disagreed or aggreed with the prompt.
```
Comment (apluscs) [inline src/main/resources/templates/email/feedbackToInterviewee.txt]: Nit: aggreed --> agreed, same for the other file.

Comment (apluscs) [inline src/main/resources/templates/email/feedbackToInterviewee.txt]: Nit: "strongly disagreed(1) or agreed(10)" just to be clear

**Review by apluscs — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 1 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt]: Done

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 2 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt]: Done

Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt]: Done

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+The values 1-10 represent whether your interviewer strongly disagreed(1) or agreed(10) with the prompt. 
+
+The interviewee communicated their thought process as they went along: {{question_1}}
+The interviewee understood the time complexity of their solution: {{question_2}}
+The interviewee took the time to consider better solutions: {{question_3}}
+The interviewee came up with an example that they used to test their solution: {{question_4}}
+The interviewee listed a handful of edge cases and accounted for their behaviour: {{question_5}}
+The interviewee asked clarifying questions: {{question_6}}
+This InterviewMe experience was helpful: {{question_7}}
```
Comment (apluscs) [inline src/main/resources/templates/email/feedbackToInterviewee.txt:11]: lf we're going with inputs on the next line on the form we should do that here too. So:
[statement]
[answer]
[new line]

Same for other file.

**Review by apluscs — COMMENTED** [review]
Change depending on how PR #125 goes

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 3 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt:11]: This format reflects the current format of the forms.

**Review by apluscs — APPROVED** [review]

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+What is the interviewee's strongest skill?:
```
Comment (jalexanderqed) [inline src/main/resources/templates/email/feedbackToInterviewee.txt]: Probably omit the colon after these question marks. It looks weird.

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 5]
```diff
@@ -0,0 +1,26 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
+
+The values 1-10 represent whether your interviewer strongly disagreed(1) or agreed(10) with the prompt. 
+
+The interviewee communicated their thought process as they went along: {{question_1}}
```
Comment (jalexanderqed) [inline src/main/resources/templates/email/feedbackToInterviewee.txt:5]: Optional: it might end up making the code that fills in this template simpler if you started the numbering from zero. However, I'm not confident about that, so feel free to leave as-is.

File: src/main/resources/templates/email/feedbackToInterviewee.txt [hunk 6]
```diff
@@ -0,0 +1,26 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below.
```
Comment (jalexanderqed) [inline src/main/resources/templates/email/feedbackToInterviewee.txt]: Grammar nit: there should be a comma before "which"

**Review by jalexanderqed — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 4 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt]: Done

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 6 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt]: Done

File: src/main/resources/templates/email/feedbackToInterviewee.txt (hunk 5 above)
Comment (GrantComm) [reply src/main/resources/templates/email/feedbackToInterviewee.txt:5]: Will leave as-is for now, and will change if issues occur.

File: src/main/resources/templates/email/feedbackToInterviewer.txt [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+I felt my interviewer came prepared: {{question_6}}
+This InterviewMe experience was helpful: {{question_7}}
+
+Is there anything in particular that your interviewer did to improve your overall experience?
+{{question_8}}
+
+What is something that you think your interviewer could have done better?
+{{question_9}}
+
+
```
Comment (jalexanderqed) [inline src/main/resources/templates/email/feedbackToInterviewer.txt:19]: Extra line here. Might be intentional, but if so should also be present in the other file.

**Review by jalexanderqed — APPROVED** [review]
One more comment
